strings/strings.json

```diff
@@ -7369,6 +7369,46 @@
     "en": "Remove filter on {0}",
     "fr": "Enlever le filtre sur {0}"
   },
+  "Select": {
+    "en": "Select",
+    "fr": "Sélectionnez"
+  },
+  "Unselect": {
+    "en": "Unselect",
+    "fr": "Désélectionnez"
+  },
+  "With": {
+    "en": "With",
+    "fr": "Avec"
+  },
+  "ExpandFacetValues": {
```

Comment:
We already have the translation for `Expand`, I would use that one instead of adding a new one.